src/Microsoft.AspNet.Authentication/DataHandler/TicketDataFormat.cs

```diff
@@ -7,7 +7,8 @@ namespace Microsoft.AspNet.Authentication
 {
     public class TicketDataFormat : SecureDataFormat<AuthenticationTicket>
     {
-        public TicketDataFormat(IDataProtector protector) : base(new TicketSerializer(), protector)
+        public TicketDataFormat(IDataProtector protector)
+            : base(TicketSerializer.Default, protector)
```

Comment:
Maybe we should go the other way and just nuke the static that wasn't used...  I recently nuked all the statics in the current bits, this might be a leftover...  I don't think its very compelling as a static...